.github/workflows/pr.yml

```diff
@@ -70,6 +70,7 @@ jobs:
 
     name: Review App Job
     runs-on: ubuntu-latest
+    # needs: [build]
```

Comment:
Why was this added?

Reply:
Just to _proactively_ run the review app script regardless of the success/failure of `build` ... can easily be reverted in a future PR. 👌 
Thanks for merging. :shipit: